test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Builder/TestModels/RecursiveModel.cs

```diff
@@ -57,7 +57,7 @@ public class RecursiveAddress
         public string City { get; set; }
 
         // Navigation property
-        public CountryDetails Country { get; set; }
```

Comment:
CountryOrRegion?
Please check for other places for similar changes.

Reply:
It's ok to use the type name.

Reply:
I also think this is minor but it could possibly cause confusion when determining between the typename and property name during debug. Maybe CountryInfo?

Reply:
Thanks. There are a lot of property named as property type name. It's easy on tracking and no confusing I think. for example:

class Customer
{
       public Order Order {get;set;}
       public Color Color {get;set;}
}

Reply:
Won't fix.

Reply:
Okay makes sense to me. It is better to keep it consistent.